generator/aggregating_sponsored_dataset.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "generator/booking_dataset.hpp"
+#include "generator/generate_info.hpp"
+
+#include "std/unique_ptr.hpp"
+#include "std/vector.hpp"
+
+namespace generator
+{
+class AggregatingSponsoredDataset : public SponsoredDataset
+{
+public:
+  explicit AggregatingSponsoredDataset(feature::GenerateInfo const & info)
+  {
+    m_datasets.emplace_back(make_unique<BookingDataset>(info.m_bookingDatafileName,
+                                                        info.m_bookingReferenceDir));
+  }
+
+  ObjectId FindMatchingObjectId(FeatureBuilder1 const & e) const override;
```

Comment:
Please, add a comment about what interface these methods override. I.e.:
```c++
// SponsoredDataset overrides:
...
```
And feel free to remove blank lines between these overrides.